var/spack/repos/builtin/packages/py-espresso/package.py

```diff
@@ -44,8 +44,8 @@ class PyEspresso(CMakePackage):
     version('develop', branch='python')
     version('4.0.0', sha256='8e128847447eebd843de24be9b4ad14aa19c028ae48879a5a4535a9683836e6b')
 
-    # https://github.com/espressomd/espresso/pull/2244
-    patch('2244.patch')
+    # espressomd/espresso#2244 merge upstream
+    patch('2244.patch', when="@4.0.0")
```

Comment:
typically we do ranges here, in case someone adds a minor version bump later, but your decision

Reply:
It will be part of v4.0.1 or `@4.0.0` should be fine.

@KaiSzuttor please correct me if I am wrong.

Reply:
will be in 4.0.1